app/models/user.rb

```diff
@@ -8,11 +8,12 @@ class User < ApplicationRecord
   validates :email, presence: true, length: { maximum: 255 },
             format: { with: VALID_EMAIL_REGEX },
             uniqueness: { case_sensitive: false }
-  validates :password, presence: true, length: { minimum: 6 }
+  validates :password, presence: true, length: { minimum: 6 }, allow_nil: true
```

Comment:
How does it work together: presence: true and allow_nil: true?

Reply:
```presence: true``` doesn't give a chance to sign up without password. 
```allow_nil: true``` give a possibility to update user (name, email) info without password